typos-ranking-rules.md

```diff
@@ -0,0 +1,59 @@
+# Typo and Ranking rules
+
+This is an explanation of the default rules used in MeiliDB.
+
+First of all we must explain some terms that are used in this reading.
+
+- A query string is the full list of all the words that the end user is searching for results.
+- A query word is one of the word that composes the query string.
+
+
+
+## Typo rules
+
+The typo rules are used before sorting the documents, they are used to aggregate them, to choose which documents contains words similar to the queried words.
+
+We use a prefix levenshtein algorithm to check if words match. The only difference with a Levenshtein algorithm is that it accepts every word that **starts with the query words** too. Therefore words are accepted if they starts with or have equal length.
+
+
+
+The Levenshtein distance between two words _M_ and _P_ is called the minimum cost of transforming _M_ into _P_ by performing the following elementary operations:
```

Comment:
is called `the minimum cost of transforming M into P` by performing the following elementary operations